clang/docs/ReleaseNotes.rst

```diff
@@ -518,6 +518,7 @@ Improvements to Clang's diagnostics
 - Clang now diagnoses definitions of friend function specializations, e.g. ``friend void f<>(int) {}``.
 - Clang now diagnoses narrowing conversions involving const references.
   (`#63151: <https://github.com/llvm/llvm-project/issues/63151>`_).
+- Clang now diagnoses unexpanded packs within the template argument lists of function template specializations.
```

Comment:
Do you have a github issue that this fixes, or is this just something that never had an issue?  If it has one, please include here, else, this is fine.

Reply:
@erichkeane I just looked -- I don't see an open issue.